src/book/01-team/04-history/01-team-history.mdx

```diff
@@ -38,3 +38,66 @@ NUbots made it to the quarter-finals of the competition. Overall we played 13 ga
 | Eliminations for Quarter-Finals (Penalty Shootout) | 3      | 1      | 1   | 0    | 0    |
 | Quarter-Finals                                     | 0      | 1      | 0   | 0    | 1    |
 | Drop-Ins                                           | -5     | 8      | 4   | 1    | 3    |
+
+## 2010: Singapore
+
+**Major hardware changes**
+
+- NAOv3: Intel ATOM 1.6GHz, 1GB Ram,
+
+**Major software changes**
+
+** Performance and Results**
+NUBots were unfortunately knocked out during second round robins, due to a loss with rUNSWift.
+
+| Round              | Points | Played | Won | Drew | Lost |
+| ------------------ | ------ | ------ | --- | ---- | ---- |
+| First Round Robin  | 6      | 2      | 2   | 0    | 0    |
+| Second Round Robin | 2      | 3      | 0   | 2    | 1    |
+
+## 2009: Graz, Austria
+
+**Major hardware changes**
+
+- Aldebaran NAOs get an upgrade (NAOv2), Dual Non-overlapping Cameras, so they can see their feet without having to bend over..
+
+**Major software changes**
+
+- Code Purge, Remove redudent AIBO Code realised by NUPlatform
+- New Git Repository, moved away from CVS
+- Introduction of Threading for NAOs, (SeeThinkThread: 30Hz, SenseMoveThread: 100Hz).
+- Blackboard based architecture
+- Soft Colour Classifications: A transition colour, that that can belong in different categories of objects, that is assigned an object with context at a later stage in the processing pipeline.
+- Advances in Line Detection, with the introduction of corner point (L and T), penalty spot, and center circle detection (ellipse fitting).
```

Comment:
> Advances in Line Detection, with the introduction of corner point (L and T),

What is `(L and T)`?

Reply:
It's the shape of the corner lines: 

![image](https://user-images.githubusercontent.com/5924865/107141054-6668c180-697a-11eb-9650-4bc7f1f5cc32.png)

Reply:
Is that something I should have known? should we add that to the glosssary?

Reply:
I can make it clearer when I'm improving it. "Intersecting field-lines that form a corner point with the shape of an 'L', and an Intersecting field-lines which intersects in the shape of a 'T'".

FYI, I met up former members Kenny Hong, and Steven Nicklin today, they will be able to fill in more of these Gaps, from 2004-2013. They are going to give some more information, team photos, etc. So this page will grow.

Reply:
That's great news!